src/common/darktable.c

```diff
@@ -1462,6 +1462,120 @@ void dt_show_times_f(const dt_times_t *start, const char *prefix, const char *su
   }
 }
 
+static inline int dt_get_num_atom_cores()
```

Comment:
static routines should start with `_`   I understand that this convention is violated in many places but better fix them one possible.